src/test/java/automation/utils/WaitForElement.java

```diff
@@ -0,0 +1,20 @@
+package automation.utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WaitForElement {
+
+    public static void wait(WebDriver driver, By pathToElement) {
```

Comment:
If you make this class non-static, you can initialize it once in BeforeAll() and not to pass driver each time you call it's methods.

Reply:
I made this non-static but moved initializing object to @BeforeEach, because @BeforeAll have to be static method.